Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -0,0 +1,31 @@
+﻿l_english: 
+ BTWK_story_event.0001.t:0 "Uther's Sickness"
+ BTWK_story_event.0001.desc:0 "You, Uther Pendragon, were once a mighty and well respected king in years past. However, thanks to a bout of illness, you have been reduced to shadow of your former self. Managing the realm, something that you would otherwise excel at, has become an impossible task in your newfound fraility. It's not uncommon you find yourself bedridden for days at a time coping with a high fever. \n\nYou can't shake the feeling that you are not long for this world, yet you could do with so much more time. Your one and only heir is your newborn grandson, Mordred, after your only legitmate son (of the same name) was killed in battle some months prior. The babe's crippled leg - a sure sign of a bad omen - only helps to sow the seeds of discontent. Without sufficient preparation for your death, and his ascension to rule, this surely will spell disaster for the realm. \n\nWhile you would have preferred to have been Mordred's guiding hand until he was fit to rule, you know it is not to be. \n\nInstead, you choose three men as his protectors, whom you have great respect and trust for - King Tewdric of Gwent, Owain (one of your greatest champions), and the grand druid Merlin. Despite your objections, however, Merlin will only accept the role on one condition..."
```

Comment:
```suggestion
 BTWK_story_event.0001.desc:0 "You, Uther Pendragon, were once a mighty and well respected king in years past. However, thanks to a bout of illness, you have been reduced to shadow of your former self. Managing the realm, something that you would otherwise excel at, has become an impossible task in your newfound fraility. It's not uncommon you find yourself bedridden for days at a time coping with a high fever. \n\nYou can't shake the feeling that you are not long for this world, yet you could do with so much more time. Your one and only heir is your newborn grandson, Mordred, after your only legitimate son - his namesake - was killed in battle some months prior. The babe's crippled leg - a bad omen - only helps to sow the seeds of discontent. Without sufficient preparation for your death, and his ascension to rule, this surely will spell disaster for the realm. \n\nWhile you would have preferred to have been Mordred's guiding hand until he was fit to rule, you know it is not to be. \n\nInstead, you choose three men as his protectors, whom you have great respect and trust for - King Tewdric of Gwent, your champion Owain, and the grand druid Merlin. Despite your objections, however, Merlin will only accept the role on one condition..."
```